build_latest.sh

```diff
@@ -215,16 +215,13 @@ function build_image() {
 	repo=$1; shift;
 	build=$1; shift;
 	btype=$1; shift;
+    local tag=$1; shift;
 
-	local local_tags=("$@") # copy arguments to local array
-	for i in "${!local_tags[@]}"
-	do
-		tags="${tags} -t ${repo}:${local_tags[$i]}"
-	done
+	tags=" -t ${repo}:${tag}"
```

Comment:
"Is the tags var even referenced anymore?

Reply:
Yeah at the time when we call `check_build_needed` [here](https://github.com/bharathappali/openjdk-docker/blob/fix-local-vars/build_latest.sh#L228) I do accept that with current changes we can pass `expanded_tags` instead of tags but the `check_build_needed` expects string to have format ` -t repo:tag` but `expanded_tags` has a format of `-t repo:tag` if we update the `check_build_needed` not to consider space when formating `adopt_image_tag` we can completely remove the tags var here.

Reply:
Your call - I think either leave a comment explaining why they're different or make that change as you suggest.